playwright/pages/dialogs/exportGiftCardsDialog.ts

```diff
@@ -0,0 +1,15 @@
+import { Page } from "@playwright/test";
+
+export class ExportGiftCardsDialog {
+  readonly page: Page;
+
+  constructor(page: Page, readonly submitButton = page.getByTestId("submit")) {
+    this.page = page;
+  }
+
+  async exportGiftCardCodes(fileExtension: string) {
+    const fileExtensionCheckbox = this.page.getByTestId(fileExtension);
+    await fileExtensionCheckbox.click();
+    await this.submitButton.click();
```

Comment:
Let's wait after clicking for the dialog to be gone before moving forward, WDYT?
`await this.submitButton.waitFor({state:'invisible'});`